automated/android/multinode/tradefed/example-job-template-cts.yaml

```diff
@@ -0,0 +1,204 @@
+job_name: MutliNode_xTS_template
+timeouts:
+  job:
+    hours: 2
+priority: medium
+visibility: public
+reboot_to_fastboot: false
+
+protocols:
+  lava-lxc:
+    master:
+      name: lxc-xts-master
+      template: debian
+      distribution: debian
+      release: stretch
+    worker:
+      name: lxc-xts-worker
+      template: debian
+      distribution: debian
+      release: stretch
+  lava-multinode:
+    # There must be one master and an arbitrary number of additional workers, so
+    # that 1+n devices will be available in the TradeFed shell.
+    roles:
+      master:
+        count: 1
+        device_type: # TODO (a Android device, e.g., nexus4)
+        timeout:
+          minutes: 30
+      worker:
+        expect_role: master
+        host_role: master
+        count: 2
+        device_type: # TODO (a Android device, e.g., nexus4)
+        timeout:
+          minutes: 30
+
+actions:
+- deploy:
+    namespace: tlxc
+    to: lxc
+    os: debian
+    role:
+    - master
+    - worker
+    timeout:
+      minutes: 10
+    packages:
+    - adb
+    - fastboot
+    - unzip
+    - wget
+    - zip
+
+- boot:
+    namespace: tlxc
+    role:
+    - master
+    - worker
+    prompts:
+    - 'root@(.*):/#'
+    timeout:
+      minutes: 5
+    method: lxc
+
+# TODO: Device-type specific deployment.
+# - deploy:
+#     namespace: droid
+#     to: fastboot
+#     role:
+#     - master
+#     - worker
+#     timeout:
+#       minutes: 30
+#     images:
+#       # TODO
+#     os: debian
+
+- boot:
+    namespace: droid
+    role:
+    - master
+    - worker
+    timeout:
+      minutes: 10
+    method: fastboot
+
+- test:
+    namespace: tlxc
+    role:
+    - master
+    - worker
+    timeout:
+      minutes: 20
+    definitions:
+    - repository: https://review.linaro.org/qa/test-definitions
```

Comment:
Can you please replace repository url with https://github.com/Linaro/test-definitions ?